client/me/help/help-happiness-engineers/index.jsx

```diff
@@ -1,57 +1,49 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' );
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import React from 'react';
+import { shuffle } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var HappinessEngineersStore = require( 'lib/happiness-engineers/store' ),
-	HappinessEngineersActions = require( 'lib/happiness-engineers/actions' ),
-	FormSectionHeading = require( 'components/forms/form-section-heading' ),
-	Gravatar = require( 'components/gravatar' );
-
-module.exports = React.createClass( {
-	displayName: 'HelpHappinessEngineers',
-
-	mixins: [ PureRenderMixin ],
-
-	componentDidMount: function() {
-		HappinessEngineersStore.on( 'change', this.refreshHappinessEngineers );
-		HappinessEngineersActions.fetch();
-	},
-
-	componentWillUnmount: function() {
-		HappinessEngineersStore.removeListener( 'change', this.refreshHappinessEngineers );
-	},
-
-	getInitialState: function() {
-		return {
-			happinessEngineers: HappinessEngineersStore.getHappinessEngineers()
-		};
-	},
-
-	refreshHappinessEngineers: function() {
-		this.setState( { happinessEngineers: HappinessEngineersStore.getHappinessEngineers() } );
-	},
-
-	render: function() {
-		if ( this.state.happinessEngineers.length === 0 ) {
-			return null;
-		}
-		return (
-			<div className="help-happiness-engineers">
-				{ this.translate( '{{headline}}We care about your happiness!{{/headline}}{{p}}They don\'t call us Happiness Engineers for nothing. If you need help, we\'re here for you!{{/p}}', {
+import FormSectionHeading from 'components/forms/form-section-heading';
+import Gravatar from 'components/gravatar';
+import QueryHappinessEngineers from 'components/data/query-happiness-engineers';
+import { getHappinessEngineers } from 'state/happiness-engineers/selectors';
+
+function HelpHappinessEngineers( props ) {
```

Comment:
How do you feel about destructuring in the arguments to make the `props` references in the function a bit more concise?

```js
function HelpHappinessEngineers( { translate, happinessEngineers } ) {
```